nix/modules/ml.nix

```diff
@@ -27,45 +28,46 @@
     };
   };
 
-  nonReinstallablePkgs = [
```

Comment:
IIRC I originally added these only so that Hasktorch would build, back when we first began the Hasktorch integration. However, it seems that it no longer builds with `nonReinstallablePkgs` specified. Removing it entirely works